homeassistant/components/vesync/light.py

```diff
@@ -51,34 +57,149 @@ def _async_setup_entities(devices, async_add_entities):
     async_add_entities(entities, update_before_add=True)
 
 
-class VeSyncDimmerHA(VeSyncDevice, LightEntity):
-    """Representation of a VeSync dimmer."""
+class VeSyncBaseLight(VeSyncDevice, LightEntity):
+    """Base class for VeSync Light Devices Representations."""
 
-    def __init__(self, dimmer):
-        """Initialize the VeSync dimmer device."""
-        super().__init__(dimmer)
-        self.dimmer = dimmer
+    def __init__(self, device):
+        """Initialize the VeSync LightDevice."""
+        super().__init__(device)
+        self.device = device
```

Comment:
Since the parent class sets this, you can drop setting it here

Reply:
removed on last commit